source/partials/_request_example_r.md.erb

```diff
@@ -0,0 +1,14 @@
+```r
+library(httr)
+
+url = "https://ees-public-api-mock.vercel.app/api/v1/data-sets/dataSetId/query"
+<% if body_json %>
+body = '<%= body_json %>'
+<% end %>
+
+response = httr::<%= http_method.upcase %>(url<% if body_json %>,
+  body=body, encode='json',
+  content_type("application/json"),
+  add_headers("Accept"= "application/json")<% end %>)
```

Comment:
Don't think we need the `Accept` header anymore. The API shouldn't throw validation errors anymore if it's not been set.

Reply:
Yep, works without that line now. Removed it in the latest commit.